CHANGELOG.md

```diff
@@ -0,0 +1,451 @@
+
+# Changelog
+
+## Future Release
+
+Some highlights to be included in the next release:
+
+* Bump to chisel 3.5.4 (https://github.com/chipsalliance/rocket-chip/pull/3105)
+
+* As Scala FIRRTL Compiler (SFC) is being sunset, update chisel2 legacy code to chisel3 (https://github.com/chipsalliance/rocket-chip/pull/3097)

```

Comment:
Both of these are important coming changes, but technically the removal of Chisel 2 compatibility mode is unrelated to the sunsetting of SFC, they're just both slated to be marked deprecated in Chisel 3.6.